rotkehlchen/chain/ethereum/modules/aave/v2/accountant.py

```diff
@@ -0,0 +1,46 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures.base import get_tx_event_type_identifier
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.accounting.interfaces import ModuleAccountantInterface
+from rotkehlchen.chain.ethereum.accounting.structures import TxEventSettings
+
+from ..constants import CPT_AAVE_V2
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+
+
+class Aavev2Accountant(ModuleAccountantInterface):
+    def event_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSettings]:  # pylint: disable=unused-argument  # noqa: E501
+        """Being defined at function call time is fine since this function is called only once"""
+        return {
+            get_tx_event_type_identifier(HistoryEventType.DEPOSIT, HistoryEventSubType.DEPOSIT_ASSET, CPT_AAVE_V2): TxEventSettings(  # noqa: E501
+                taxable=False,
+                count_entire_amount_spend=False,
+                count_cost_basis_pnl=False,
+                method='spend',
+                take=1,
+            ),
+            get_tx_event_type_identifier(HistoryEventType.WITHDRAWAL, HistoryEventSubType.REMOVE_ASSET, CPT_AAVE_V2): TxEventSettings(  # noqa: E501
+                taxable=False,
+                count_entire_amount_spend=False,
+                count_cost_basis_pnl=False,
+                method='acquisition',
+                take=1,
+            ),
```

Comment:
checking the maker module there we account for a lost on the extra amount that you paid when withdrawing the original assets. Can you also add that check here? In that case we do it checking the vault but here I believe is feasible by keeping track of the assets you provide vs the assets you return as we do in the maker module. Probably in the transactions that you have as example you already see this. Let me know of any issue regarding this topic